### ISSUE TITLE
Please adjust versionCode & versionName on releases

### ISSUE THREAD

**Issue by IzzySoft** [tagspaces/tagspaces#1313]
It seems all your releases for Android APKs identify as the exactly same version:

    package: name='org.tagspaces.mobile' versionCode='10000' versionName='1.0.0' compileSdkVersion='29' compileSdkVersionCodename='10'

As Android uses `versionCode` to tell versions apart that means, once a user has installed a version, they never get any update information (as Android considers the next version being the same one that's already installed). Further, as you neither change `versionName`, users cannot even tell which version they have installed.

Above string is shown by `aapd d badging` for the beta-3.8.1 as well as for the 3.8.4. Both files differ (i.e. they are not identical). For the latter one e.g. would expect `versionCode='3.8.4'`.

**TL;DR:** can you please always increase `versionCode` when creating releases – and set `versionName` to the real version?

**Comment by uggrock** [comment]
Thanks for pointing this out, we will adjust the build scripts accordingly.

**Comment by IzzySoft** [comment]
@uggrock did you change the package name intentionally with v3.9.8? Before it was `org.tagspaces.mobile`, now it suddenly is `org.tagspaces.mobileapp` (and thus does not work as update, but is an entirely new app). Can you please tell me which of the two will persist with future releases?

**Comment by uggrock** [comment]
@IzzySoft yes the package name was changed in the 3.9.6 release. The Android app with the org.tagspaces.mobile package name was deactivated by Google, because of a duplication of content reason. We were having two apps in Google Play one for 2.x release and one for 3.x release. And this is apparently against the Google policies, so both apps were suspended by Google and we were forced to publish the app again under new name and package name.

**Comment by IzzySoft** [comment]
Google and their crazy rules… Sad to read. Thanks for explaining! Might be worth mentioning that [on the release details](https://github.com/tagspaces/tagspaces/releases/tag/v3.9.8), guess I'm not the only one confused :wink:
